### PR TITLE
[ansible]add upgrade node_exporter playbook

### PR DESCRIPTION
**Pull request by jhd2best**
the playbook which can batch upgrade node_exporter in mainnet internal node

### CODE REVIEW

**Comment by jhd2best** [comment]
**Test Result**
PASS

**Test Logs**
ansible-playbook -i inventory/test.hosts playbooks/upgrade-node-exporter.yml -e 'inventory=test user=hmy'

PLAY [test] ********************************************************************************

TASK [Gathering Facts] *********************************************************************
ok: [52.\*.\*.139]

TASK [gather OS specific variables] ********************************************************
ok: [52.\*.\*.139] => (item=ansible/playbooks/vars/Azure-CentOS.yml)

TASK [download latest install package] *****************************************************
changed: [52.\*.\*.139]

TASK [tar install package] *****************************************************************
changed: [52.\*.\*.139]

TASK [install node_exporter] ***************************************************************
changed: [52.\*.\*.139]

TASK [change owner of bin] *****************************************************************
changed: [52.\*.\*.139]

TASK [restart node_exporter service] *******************************************************
changed: [52.\*.\*.139]

TASK [clear trash] *************************************************************************
changed: [52.\*.\*.139] => (item=/home/hmy/node_exporter-1.0.1.linux-amd64.tar.gz)
changed: [52.\*.\*.139] => (item=/home/hmy/node_exporter-1.0.1.linux-amd64)

PLAY RECAP *********************************************************************************
52.\*.\*.139             : ok=8    changed=6    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0

File: playbooks/upgrade-node-exporter.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+- hosts: "{{ inventory }}"
+  remote_user: '{{ user }}'
+  serial: 1
```
Comment (LeoHChen) [inline playbooks/upgrade-node-exporter.yml:4]: no need do serial: 1 for node exporter upgrade

File: playbooks/upgrade-node-exporter.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+- hosts: "{{ inventory }}"
+  remote_user: '{{ user }}'
```
Comment (LeoHChen) [inline playbooks/upgrade-node-exporter.yml:3]: no need to specify remote_user now, we can use ansible_user on the inventory. that's easier.

Comment (jhd2best) [reply playbooks/upgrade-node-exporter.yml:3]: sure

**Pull request merged**